src/main/webapp/WEB-INF/tags/page.tag

```diff
@@ -0,0 +1,23 @@
+<%@ tag description="Generic TEAMMATES Page" %>
+<%@ taglib tagdir="/WEB-INF/tags" prefix="t" %>
+<%@ attribute name="pageTitle" %>
+<%@ attribute name="headerJs" fragment="true" %>
+<%@ attribute name="navBar" fragment="true" %>
+<%@ attribute name="bodyTitle" %>
+<!DOCTYPE html>
+<html>
+<head>
+    <title>${pageTitle}</title>
+    <t:includes />
+    <jsp:invoke fragment="headerJs" />
+    <t:enableJS />
+</head>
+<body>
+    <jsp:invoke fragment="navBar" />
+    <div id="frameBodyWrapper" class="container theme-showcase">
+        <t:header title="${bodyTitle}" />
+        <jsp:doBody />
+    </div>
+    <t:footer version="tempVal"/>
```

Comment:
It originally uses a reference to the static Config class which is impossible in JSTL. I can set it using scriptlets, but I honestly think it's better settling on a good method to handle all constants beforehand.
Hence, it's temporarily a tempVal.